metaflow/plugins/cards/ui/src/App.svelte

```diff
@@ -3,6 +3,7 @@
   import "./prism";
   import "./global.css";
```

Comment:
do we need this at all now?

Reply:
This is for rules at the index.html level, which we still have. I left the variables in there too